src/app/Library/Uploaders/Validation/ValidBackpackUpload.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\Uploaders\Validation;
+
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanelFacade;
+use Closure;
+use Illuminate\Contracts\Validation\DataAwareRule;
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Contracts\Validation\ValidatorAwareRule;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rules\File;
+
+class ValidBackpackUpload implements ValidationRule, DataAwareRule, ValidatorAwareRule
+{
+    /**
+     * @var \Illuminate\Contracts\Validation\Validator
+     */
+    protected $validator;
+
+    protected array $data;
+
+    public array $arrayRules = [];
+
+    public array $fileRules = [];
+
+    public ?Model $entry;
+
+    public static function make(): self
+    {
+        $instance = new static();
+        $entry = CrudPanelFacade::getCurrentEntry();
+        $instance->entry = $entry !== false ? $entry : null;
+
+        return $instance;
+    }
+
+    /**
+     * Run the validation rule.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @param  Closure(string): \Illuminate\Translation\PotentiallyTranslatedString  $fail
+     * @return void
+     */
+    public function validate(string $attribute, mixed $value, Closure $fail): void
+    {
+    }
```

Comment:
Why does this method exist but is empty? 👀

Reply:
Just so that I knew the signature when working 🙃 I will delete it, it's the overwrite from the parent class.